ui/app/AppLayouts/Chat/ChatColumn/Message.qml

```diff
@@ -153,6 +156,17 @@ Item {
         }
     }
 
+    Connections {
+        target: chatsModel
+        onHideMessage: {
+            // This hack is used because message_list deleteMessage sometimes does not remove the messages (there might be an issue with the delegate model)
```

Comment:
This is very weird. I left a message about it in delete, I hope it fixes it.

Reply:
:'( it did not